test/hotspot/jtreg/runtime/cds/appcds/LotsOfSyntheticClasses.java

```diff
@@ -37,7 +37,7 @@
  * @summary Try to archive lots and lots of classes.
```

Comment:
The copyright header "Copyright Amazon.com Inc" miss the year message.

Reply:
This is as designed.